dev_guide/routes.adoc

```diff
@@ -164,3 +164,47 @@ console or using `oc get routes` displays the host and certificates specified by
 the user. However, custom hosts are not respected, and routes with custom
 certificates do not work.
 endif::[]
+
+[[routes-load-balancing-for-AB-testing]]
+== Load Balancing for A/B Testing
+You can run two versions of an application and control,
```

Comment:
It would be good to give a short blurb here on what A/B testing is and how it benefits the customer now that OpenShift supports it.